### DISCUSSION TITLE
pFrozenColumn not working properly with *ngIf condition

### DISCUSSION THREAD

**Discussion by Sapna-Kale** [primefaces/community#815] [PrimeNG]
Hi,

I have 4 fixed columns, 1 is with *ngIf condition. the column with ngIf is not getting frozen (left is not calculated properly)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
